_config/site.yml

```diff
@@ -52,7 +52,7 @@ profile_base: &profile
 profiles:
   development: &development
     dcp_base_url: http://dcpbeta-searchisko.rhcloud.com/
-    dcp_base_protocol_relative_url: //dcpbeta-searchisko.rhcloud.com/
+    dcp_base_protocol_relative_url: //dcp2-searchisko.rhcloud.com/
```

Comment:
This will make the whole site use DCP2. You just want the projects search to use that. See my PR for moving Buzz to DCP2